src/tests/ftest/util/ior_test_base.py

```diff
@@ -150,18 +144,15 @@ def run_ior_with_pool(self, intercept=None, test_file_suffix="",
         # start dfuse if api is POSIX
         if self.ior_cmd.api.value == "POSIX":
             # Connect to the pool, create container and then start dfuse
-            # Uncomment below two lines once DAOS-3355 is resolved
-            if self.ior_cmd.transfer_size.value == "256B":
-                return "Skipping the case for transfer_size=256B"
-            self._start_dfuse()
+            self._start_dfuse(debug)
             test_file = os.path.join(self.dfuse.mount_dir.value, "testfile")
         elif self.ior_cmd.api.value == "DFS":
             test_file = os.path.join("/", "testfile")
 
         self.ior_cmd.test_file.update("".join([test_file, test_file_suffix]))
 
-        out = self.run_ior(self.get_ior_job_manager_command(), self.processes,
-                           intercept)
+        out = self.run_ior(self.get_job_manager_command(), self.processes,
```

Comment:
(pylint-too-many-function-args) Too many positional arguments for method call
(pylint-no-member) Instance of 'IorTestBase' has no 'get_job_manager_command' member